proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,125 +44,73 @@ direction of the policies set by the Board.
+
+* Regular members
+* Voting members
+
+**Regular members** Regular members have the roles and responsibilities
+as outlined below. In any case where it is not specified that a role
+or responsibility applies to a Voting member, then a Regular member is
+implied. Any member of one of the projects under the foundation may request to
+become a regular member by opening a PR to add themselves to the list of
+regular members. Regular members remain for as long as they are active
+within the work of the CPC. Regular members who have not been active
+in GitHub, participated in meetings, or other work of the CPC for 3 months may be
+removed from the list of Regular members. In addition, a Regular CPC member
+can be removed by by voluntary resignation, or by a standard CPC motion.
+
+**Voting members** Voting members have the roles and responsibilities
+as outlined below. Voting members are selected as follows:
+
+* Each top level project many nominate 2 member through a process
+of their choosing. Once nominated the member must be ratified by the
+board before becoming a Voting member.
+
+* 2 Voting member may be nominated by the projects excluding the
+top level projects. If a vote is required amoung candidates
+nominated by different projects, each of the non-top level projects
+will have 2 votes (one for each position).
+
+* 2 Voting members may be nominated by the Regular members. Once
+nominated these members must be ratified by the board before
+becoming a Voting member.
+
+Voting members are expected to make a time commitment that
+allows them to be responsive to CPC business, participate regularly
+in meetings and to participate in all voting matters.  They are
+also expected to help to "enable" work of the regular CPC
+members by providing leadership, help with interactions with the
+board and Foundation staff and to generally help grease the
+wheels to keep things moving.
+
+Voting members serve for a term of 1 year and must be re-nominated
+and ratified by the board each year. In addition, a Voting CPC member
+can be removed by by voluntary resignation, by a standard CPC motion,
+or in accordance to the participation rules described for Regular
+members.
+ 
 Changes to CPC membership should be posted in the agenda, and may be suggested
 as any other agenda item.
 
-No more than one-fourth of the CPC members may be affiliated with the
-same employer. If removal or resignation of a CPC member, or a change of
-employment by a CPC member, creates a situation where more than
-one-fourth of the CPC membership shares an employer, then the situation
+No more than one-fourth of the Voting CPC members may be affiliated with the
+same employer. If removal or resignation of a Voting CPC member, or a change of
+employment by a Voting CPC member, creates a situation where more than
+one-fourth of the Voting CPC membership shares an employer, then the situation
 must be immediately remedied by the resignation or removal of one or more
 PC
```

Comment:
Considering that CPC voting members are elected by projects, should we give projects the chance to elect another member before resignation? I would like to avoid the situation where a project has both his voting members out because of this.

Reply:
I'm not sure we can stop somebody from resigning other than asking them to provide some time for a new election.

Reply:
I think a top level project should always have 2 voting members. Up until somebody new is elected by the project, the previous member is still voting.